service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DeserializeAndAcceptEmptyStringsAsNullObjectTest.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+import java.io.File;
+import java.io.IOException;
+
+@ActiveProfiles("integration-test")
+@WebMvcTest(DeserializeAndAcceptEmptyStringsAsNullObjectTest.class)
+@OverrideAutoConfiguration(enabled = true)
+@SuppressWarnings("unchecked")
```

Comment:
unnecessary

Reply:
i've removed supress warnings but it fails if remove the other two

Reply:
Yeah, just the suppress was all I meant, could have been clearer there